joplin/templates/wagtailadmin/pages/listing/_table_headers_search.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n %}
```

Comment:
we arent using this at all, i just stubbed it hear to stash the changes @robbobfrh84 made. 

I'm not super clear on why or if they are needed, but if they are, I would like to have a new template partial specifically for search or otherwise differentiate between other times the explore template might get used.